seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonError.java

```diff
@@ -199,4 +210,19 @@ public static SeaTunnelRuntimeException sqlTemplateHandledError(
         params.put("optionName", optionName);
         return new SeaTunnelRuntimeException(SQL_TEMPLATE_HANDLED_ERROR, params);
     }
+
+    public static SeaTunnelRuntimeException unsupportedArrayGenericType(
+            String identifier, String dataType) {
+        Map<String, String> params = new HashMap<>();
+        params.put("identifier", identifier);
+        params.put("dataType", dataType);
+        return new SeaTunnelRuntimeException(UNSUPPORTED_ARRAY_GENERIC_TYPE, params);
+    }
+
+    public static SeaTunnelRuntimeException unsupportedRowKind(String identifier, String rowKind) {
```

Comment:
we need tableId when invoke `unsupportedRowKind` method.

Reply:
> we need tableId when invoke `unsupportedRowKind` method.

OK, I will optimize the code based on your suggestions

Reply:
> > we need tableId when invoke `unsupportedRowKind` method.
> 
> OK, I will optimize the code based on your suggestions

done.